lib/unbeatable_computer.rb

```diff
@@ -1,11 +1,13 @@
+require_relative 'marks'
+
 class UnbeatableComputer
 
   def initialize(ui)
     @ui = ui
   end
 
-  MARKS = {:computer => "O",
-           :opponent => "X"
+  MARKS = {:computer => Marks::OPPONENT,
```

Comment:
Having the name `opponent` and `user` is a little confusing. Is the computer always an opponent? What if the computer goes first, then is the human the opponent? Would calling them Marks::X and Marks::O be clearer?

Reply:
Yes!! OMG, I can't believe I couldn't think that!